bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -105,6 +102,8 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
           <p>Yes, here it is:
             <a href="https://wiki.mozilla.org/WebAppSec/Web_App_Severity_Ratings">
               Web Application Severity Ratings</a></p>
+              The bounty committee's reward is based on the security and privacy 
+              impact of the bug and ultimately the importance of the website or service. 
```

Comment:
I thought it was more "the sensitivity of the information handled by the website or service", rather than its "importance".